contracts/modules/cover/CoverOrder.sol

```diff
@@ -0,0 +1,225 @@
+// SPDX-License-Identifier: MIT
```

Comment:
1. This is not an internal contract, this should be moved to `contracts/external/`, maybe together with cover broker.
2. Why `MIT` and not `GPL-3.0-only`?